src/GameLogic.cpp

```diff
@@ -213,7 +236,7 @@ bool GameLogic::setScore(int level, int score) {
 }
 
 void GameLogic::resetScores() {
-    for (int i = 0; i < scores.size(); i++) {
+    for (int i = 0; i < scores.size(); ++i) {
```

Comment:
There was no reason why you needed to change this

Reply:
It was just to match all the other for loops, just looks better